tools/autograph-monitor/monitor.go

```diff
@@ -65,6 +71,16 @@ func main() {
 		log.Fatal("AUTOGRAPH_KEY must be set to the api monitoring key")
 	}
 
+	ignoredExpSignerIdsVar := os.Getenv("IGNORE_EXPIRATION_SIGNER_IDS")
```

Comment:
IMO best practices for adding new env var in code is to always add the env in the corresponding config file, even if it's not used. Will you create corresponding ENV var with empty value in the autograph-config repo?